validators/RequestSchema.js

```diff
@@ -11,6 +11,23 @@ const ItemSchema = FileSchema.keys({
   children: Joi.array().items(Joi.string()),
 })
 
+// Homepage
+const UpdateHomepageSchema = Joi.object({
+  content: Joi.object({
+    frontMatter: Joi.object({
+      layout: Joi.string().required(),
+      title: Joi.string().required(),
+      description: Joi.string(),
```

Comment:
will the FE allow a case like so: `description: ""`?

Reply:
Good catch, that should be allowed - fixed in https://github.com/isomerpages/isomercms-backend/pull/331/commits/1b04b913d20b907d1b8d2ad1f6dc8f496017cbe5